runtime/development/src/lib.rs

```diff
@@ -1966,6 +1969,7 @@ construct_runtime!(
 		BaseFee: pallet_base_fee::{Pallet, Call, Config<T>, Storage, Event} = 162,
 		Ethereum: pallet_ethereum::{Pallet, Config, Call, Storage, Event, Origin} = 163,
 		EthereumTransaction: pallet_ethereum_transaction::{Pallet, Storage, Event<T>} = 164,
+		ConnectorsAxelarGateway: connectors_gateway_axelar_precompile::{Pallet, Call, Storage, Event<T>} = 165,
```

Comment:
Is it no longer possible to place this together with the other connectors pallet?

Reply:
I thought it belongs to EVM stuff. But can move

Reply:
Sorry, I misread it as "ConnectorsGateway" 🙃  all good, sorry for the noise!